CMakeLists.txt

```diff
@@ -3,14 +3,72 @@ project (nuQoin)
 # detect operating system
 message(STATUS "We are on a ${CMAKE_SYSTEM_NAME} system")
 
+find_package(Boost COMPONENTS filesystem system unit_test_framework) 
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(JSONCPP jsoncpp)
+link_libraries(${JSONCPP_LIBRARIES})
+include_directories(${JSONCPP_INCLUDE_DIRS})
+
+###################
+
+message(STATUS "checking for module 'jsoncpp'")
```

Comment:
❓ When building on Windows, I did not find that this was required. Do you know where it's used?

Reply:
Yes, the next bit with find_library(JSONCPP_LIBRARY NAMES libjsoncpp.a libjsoncpp.dylib... suffices. This part can be removed.

Reply:
💭 I don't think I have libjsoncpp at all. Are you sure it's not the json_spirit library getting used?